stable/traefik/templates/deployment.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicas }}
```

Comment:
Sure, I only realised this after. Due to the way user settings are merged in with the default `values.yaml`, I think generally defaults have to live in the templates anyway.